lib/rules/prefer-explicit-assert.ts

```diff
@@ -55,14 +97,33 @@ export default createTestingLibraryRule<Options, MessageIds>({
   create(context, [options], helpers) {
     const { assertion } = options;
     const getQueryCalls: TSESTree.Identifier[] = [];
+    const findQueryCalls: TSESTree.Identifier[] = [];
 
     return {
       'CallExpression Identifier'(node: TSESTree.Identifier) {
         if (helpers.isGetQueryVariant(node)) {
           getQueryCalls.push(node);
         }
+
+        if (helpers.isFindQueryVariant(node)) {
+          findQueryCalls.push(node);
+        }
       },
       'Program:exit'() {
+        findQueryCalls.forEach((queryCall) => {
+          if (isVariableDeclaration(queryCall)) {
```

Comment:
Wouldn't be better to exclude those these nodes when pushing them to `findQueryCalls`?

Reply:
Not sure about that 🤔 Now there's even more logic in this if statement, so technically we could move everything into `'CallExpression Identifier'(node: TSESTree.Identifier) {`, but I think the current version is more consistent with how `getBy` case is implemented.

Reply:
Fair enough, we can leave it as it is.